autogpt_platform/frontend/tailwind.config.ts

```diff
@@ -116,11 +116,19 @@ const config = {
         96: "24rem",
       },
       borderRadius: {
+        // Design system border radius tokens from Figma
+        xs: "0.25rem", // 4px
+        s: "0.5rem", // 8px
+        m: "0.75rem", // 12px
+        l: "1rem", // 16px
+        xl: "1.25rem", // 20px
+        "2xl": "1.5rem", // 24px
+        full: "9999px", // For pill buttons
```

Comment:
I think I saw the root font size inconsistency between pages, causing the rem size change, just to keep in mind in case you find it and manage to fix it :D